### PR TITLE
Pilot mode half-pager

### PR DESCRIPTION
**Pull request by lesjonesiv**
Pilot mode guide

### CODE REVIEW

File: fern/docs/pages/pilot-mode.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+to use licensed state requirements where appropriate when assessing loan compliance.
+
+## Getting started
+In order to operate in pilot mode, you must set your contact details in order to serve as lender of record. Please see
+the [`/configuration/lending_details`](/api-reference/configuration/set-lending-details) for more information. You must
+set your details before you can run the origination flow in Pier. After that, you may use the Pier API as normal -
+see the [quickstart](/overview/quickstart) section for more information. Remember, you must handle disbursements and
+payments yourself.
+
+If you wish to upgrade the product to take advantage of Pier's funds flow and lending licenses, please get in touch.
```
Comment (mjpierfinance) [inline fern/docs/pages/pilot-mode.mdx:26]: Should we link any contact details to `please get in touch`?

**Review by mjpierfinance — APPROVED** [review]
non-blocking comment but lgtm 🚀

File: fern/docs/pages/pilot-mode.mdx (hunk 1 above)
Comment (lesjonesiv) [reply fern/docs/pages/pilot-mode.mdx:26]: Yeah, that's a good call - I thought about that myself but figured right now Jess is pretty much already in touch with everyone with sandbox keys, but let's see if there's something we should add here.

Comment (lesjonesiv) [reply fern/docs/pages/pilot-mode.mdx:26]: Waiting to hear here: https://pierspace.slack.com/archives/C04D6V2GER0/p1705972662209969

**Review by mjpierfinance — APPROVED** [review]

**Pull request merged**